lib/libvfio-user.c

```diff
@@ -1209,6 +1213,29 @@ exec_command(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg)
     return ret;
 }
 
+static int
+do_reply(vfu_ctx_t *vfu_ctx, vfu_msg_t *msg, int ret)
```

Comment:
now you factored this out, could we not re-use "ret", have the arg be "reply_ret" or something? This always caught me off guard